tests/alert-max/alert-max-append-higher-priority/test.rules

```diff
@@ -0,0 +1,17 @@
+# internal id 0 -- should not trigger
```

Comment:
This looks complex... Will it be stable ?

Reply:
Not sure I am the best one to judge, but Victor agreed that we should try to have this test as an S-V one...

Reply:
Good enough for me :-)